packages/victory-core/src/victory-util/hooks/use-animation-state.ts

```diff
@@ -4,26 +4,41 @@ import { defaults, some } from "lodash";
 import * as Collection from "../collection";
 import * as Transitions from "../transitions";
 
-const INITIAL_STATE = {
+const INITIAL_STATE: AnimationState = {
   nodesShouldLoad: false,
   nodesDoneLoad: false,
   animating: true,
 };
 
+type AnyObject = Record<string, any>;
```

Comment:
I wish I would've used a type like this more frequently in TS conversions ... basically this just means "any not null".
Thinking out-loud, is this what `{}` is for?